src/commands/junit/utils.ts

```diff
@@ -1,21 +1,14 @@
 import {lstatSync} from 'fs'
 
+import {getCommonAppBaseURL} from '../../helpers/app'
 import {SpanTags} from '../../helpers/interfaces'
 import {CI_JOB_URL, CI_PIPELINE_URL, GIT_BRANCH, GIT_REPOSITORY_URL, GIT_SHA} from '../../helpers/tags'
 
 export const getBaseUrl = () => {
-  if (process.env.DATADOG_SITE || process.env.DD_SITE) {
-    let site = process.env.DATADOG_SITE || process.env.DD_SITE
-    if (site === 'datadoghq.com') {
-      site = 'app.datadoghq.com'
-    } else if (site === 'datad0g.com') {
-      site = 'dd.datad0g.com'
-    }
```

Comment:
don't we need this staging logic?

Reply:
no, it should be covered by the common function already I think